sdk/communication/communication-sms/README.md

```diff
@@ -62,33 +63,70 @@ npm install @azure/identity
 ```
 
 The [`@azure/identity`][azure_identity] package provides a variety of credential types that your application can use to do this. The README for @azure/identity provides more details and samples to get you started.
+AZURE_CLIENT_SECRET, AZURE_CLIENT_ID and AZURE_TENANT_ID environment variables are needed to create a DefaultAzureCredential object.
```

Comment:
That's not really the case. Environment variables are only one way of how to use the `DefaultAzureCredential`